pedpy/methods/method_utils.py

```diff
@@ -567,11 +600,24 @@ def compute_crossing_frames(
         )
     )
 
-    # crossing means, the current movement crosses the line and the end point
-    # of the movement is not on the line. The result is sorted by frame number
-    crossing_frames = df_movement.loc[
-        shapely.intersects(df_movement.movement, measurement_line.line)
-    ][[ID_COL, FRAME_COL]]
+    if count_on_line:
+        crossing_frames = df_movement.loc[
+            shapely.intersects(df_movement.movement, measurement_line.line)
+        ][[ID_COL, FRAME_COL]]
+    else:
```

Comment:
Suggestion
`    # Case when crossing means movement crosses the line, but the end point is not on it.
`

Reply:
Added comment